src/main/java/com/gmail/woosay333/onlinebookstore/controller/BookController.java

```diff
@@ -19,17 +21,30 @@ public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookResponseDto> getAllBooks() {
+    public List<BookDto> getAllBooks() {
         return bookService.findAll();
     }
 
     @GetMapping("/{id}")
-    public BookResponseDto getBookById(@PathVariable Long id) {
+    public BookDto getBookById(@PathVariable Long id) {
         return bookService.findById(id);
     }
 
     @PostMapping
-    public BookResponseDto saveBook(@RequestBody CreateBookRequestDto bookRequestDto) {
+    public BookDto saveBook(@RequestBody CreateBookRequestDto bookRequestDto) {
         return bookService.save(bookRequestDto);
     }
+
+    @PutMapping("/{id}")
+    public BookDto updateBook(
+            @PathVariable Long id,
+            @RequestBody CreateBookRequestDto bookRequestDto
```

Comment:
It is now used not only for creation
```suggestion
            @RequestBody BookRequestDto bookRequestDto
```